docs/assessment.md

```diff
@@ -22,7 +22,11 @@ Each assessment is a single directory in the `assessments` folder or any subfold
 
 The assessment ID is the full path relative to `assessments`.
 
-- [Format specification for assessment `infoAssessment.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/schemas/schemas/infoAssessment.json)
+??? example "Format specification for assessment `infoAssessment.json`"
```

Comment:
The styling on the example makes it really stand out on the page, while I think in this case we don't really want to highlight the schema (most users will not want to look at this). Is there something other than "example" which has less noticeable styling? Some kind of "technical details, ignore unless you really want it"? Maybe a `note`?